src/managers/KeybindManager.cpp

```diff
@@ -567,6 +567,20 @@ int repeatKeyHandler(void* data) {
     return 0;
 }
 
+int longPressHandler(void* data) {
+    SKeybind** ppActiveKeybind = (SKeybind**)data;
+
+    if (!*ppActiveKeybind || g_pSeatManager->keyboard.expired())
```

Comment:
kinda sucks. Why not remove the timer on key up?

Reply:
no, if there is a long press pending and any key is pressed/released it should reset the timer, probably. Though idk how that will work with transient ones.... actually leave it as is I guess.

Reply:
> Though idk how that will work with transient ones
wdym? should i try implement it and check via armed() func(from CEventLoopTimer)?

Reply:
na dw